tools/msys2_fetch_and_build_all_tx16s.sh

```diff
@@ -15,81 +150,111 @@ do
 	fi
 done
 
+STEP=1
 echo "=== Step $((STEP++)): Creating a directory for EdgeTX ==="
-mkdir ~/edgetx && cd ~/edgetx
+mkdir -p ${PROJ_DIR} && cd ${PROJ_DIR}
+check_command $? "mkdir -p ${PROJ_DIR} && cd ${PROJ_DIR}"
 if [[ $PAUSEAFTEREACHLINE == "true" ]]; then
   echo "Step finished. Please check the output above and press Enter to continue or Ctrl+C to stop."
   read
 fi
 
-echo "=== Step $((STEP++)): Fetching EdgeTX source tree (main branch) from GitHub ==="
-git clone --recursive -b main https://github.com/EdgeTX/edgetx.git edgetx_main
+if [[ ! -d "${SOURCE_DIR}" ]]; then
+  echo "=== Step $((STEP++)): Fetching EdgeTX source tree (${BRANCH_NAME} branch) from GitHub ==="
+  git clone --recursive -b ${BRANCH_NAME} https://github.com/EdgeTX/edgetx.git ${SOURCE_DIR}
+  check_command $? "git clone --recursive -b ${BRANCH_NAME} https://github.com/EdgeTX/edgetx.git ${SOURCE_DIR}"
+  cd ${SOURCE_DIR}
+else
+  echo "=== Step $((STEP++)): updating EdgeTX source tree (${BRANCH_NAME} branch) from GitHub ==="
+  cd ${SOURCE_DIR}
+  check_command $? "cd ${SOURCE_DIR}"
+  git checkout ${BRANCH_NAME}
+  check_command $? "git checkout ${BRANCH_NAME}"
+  git pull
+  check_command $? "git pull"
+  git submodule update --init --recursive
+  check_command $? "git submodule update --init --recursive"
+fi
 if [[ $PAUSEAFTEREACHLINE == "true" ]]; then
   echo "Step finished. Please check the output above and press Enter to continue or Ctrl+C to stop."
   read
 fi
 
 echo "=== Step $((STEP++)): Creating build output directory ==="
-cd edgetx_main && mkdir build-output && cd build-output
+[[ -d ${BUILD_OUTPUT_DIR} ]] && rm -rf ${BUILD_OUTPUT_DIR}
+mkdir -p ${BUILD_OUTPUT_DIR} && cd ${BUILD_OUTPUT_DIR}
+check_command $? "mkdir -p ${BUILD_OUTPUT_DIR} && cd ${BUILD_OUTPUT_DIR}"
 if [[ $PAUSEAFTEREACHLINE == "true" ]]; then
   echo "Step finished. Please check the output above and press Enter to continue or Ctrl+C to stop."
   read
 fi
 
-echo "=== Step $((STEP++)): Running CMake for TX16S as an example ==="
-cmake -G "MSYS Makefiles" -Wno-dev -DCMAKE_PREFIX_PATH=$HOME/5.12.9/mingw73_64 -DSDL_LIBRARY_PATH=/mingw64/bin/ -DPCB=X10 -DPCBREV=TX16S -DDEFAULT_MODE=2 -DGVARS=YES -DPPM_UNIT=US -DHELI=NO -DLUA=YES -DCMAKE_BUILD_TYPE=Release ../
+echo "=== Step $((STEP++)): Running CMake for ${RADIO_TYPE} as an example ==="
+cmake -G "MSYS Makefiles" -Wno-dev -DCMAKE_PREFIX_PATH=$HOME/5.12.9/mingw73_64 -DSDL_LIBRARY_PATH=/mingw64/bin/ ${BUILD_OPTIONS} -DDEFAULT_MODE=2 -DGVARS=YES -DPPM_UNIT=US -DHELI=NO -DLUA=YES -DCMAKE_BUILD_TYPE=Release ../
```

Comment:
Remove the custom build options here and move them to line 14:
`cmake -G "MSYS Makefiles" -Wno-dev -DCMAKE_PREFIX_PATH=$HOME/5.12.9/mingw73_64 -DSDL_LIBRARY_PATH=/mingw64/bin/ ${BUILD_OPTIONS} -DCMAKE_BUILD_TYPE=Release ../`